guide/sensors.md

```diff
@@ -0,0 +1,53 @@
+## Sensors
+
+Webots can simulate a lot of standard robotic sensors.
+Using the nodes described [below](#generic-sensor-list) and their corresponding API is sufficient in most of the cases.
+
+More specific sensors are built on the top of these generic nodes, thanks to the [PROTO system](../reference/proto.md).
+These sensors are listed by category [in the section below](#branded-sensors).
```

Comment:
The (#branded-sensors) link is wrong. It wasn't detected by our test system?

Reply:
Strange. I'm taking a look at this.

Reply:
Fixed in: https://github.com/omichel/webots-doc/pull/737